src/domain_bridge/wait_for_graph_events.hpp

```diff
@@ -199,6 +199,11 @@ class WaitForGraphEvents
     // If there are no publishers, return an empty optional
     if (num_endpoints < 1u) {
       return {};
+    } else {
+      // If we found at least one publisher, wait for a bit (more publishers may become available)
+      std::this_thread::sleep_for(std::chrono::milliseconds(300));
```

Comment:
I would prefer to see this timeout a configurable option (e.g. maybe part of TopicBridgeOptions).

Reply:
I've added a `delay` option to `TopicBridgeOptions` in 119cf67